### PR TITLE
Rework gigasecond documentation

### PR DESCRIPTION
**Pull request by kytrinyx**
Rework gigasecond documentation

This PR is part of our project of making our Practice Exercises more consistent and human. For more context please [see this forum thread](https://forum.exercism.org/t/new-project-making-practice-exercises-more-consistent-and-human-across-exercism/3943)

We will merge this PR in roughly 72 hours, and then submit PRs to all tracks that have this exercise.

The main changes for this exercise are:
- Make it easier to understand
- Make it a bit more engaging.
- Provide more context around the idea of a gigasecond and why it is potentially interesting.
- Tone down the mathiness in an attempt to make it feel more casual, friendly, and approachable.

### CODE REVIEW

**Review by iHiD — APPROVED** [review]

**Review by BethanyG — APPROVED** [review]

**Review by ErikSchierboom — APPROVED** [review]

File: exercises/gigasecond/introduction.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- A food recipe might explain that you need to let the brownies cook in the oven for two kiloseconds (that's two thousand seconds).
+- Perhaps you and your family would travel to somewhere exotic for two megaseconds (that's two million seconds).
+- And if you and your spouse were married for _a thousand million_ seconds, you would celebrate your one gigasecond anniversary.
+
+```exercism/note
+If we ever colonize Mars or some other planet, measuring time is going to get even messier.
+If someone says "year" do they mean a year on Earth or a year on Mars?
+
+The idea of measuring time only in seconds comes from science fiction author Vernor Vinge.
+He uses the metric system as the basis for time measurements in his novel, ["A Deepness in the Sky"][vinge-novel].
```
[the remark this answers is not in the document]
Comment (kytrinyx) [reply exercises/gigasecond/introduction.md:21]: Yes, that's a good call. I've updated the text.

**Comment by habere-et-dispertire** [comment]
Raku's default implementation includes support for [leap seconds](https://github.com/rakudo/rakudo/blob/main/src/core.c/DateTime.pm6#L550) which throes the tests.

Would it be useful to mention that leap seconds are to be ignored? I know it's in the referenced [JavaScript documentation](https://262.ecma-international.org/5.1/#sec-15.9.1) for the date string as:

> In time values leap seconds are ignored. 

but it's rather buried. What do others think ?

**Comment by iHiD** [comment]
> Would it be useful to mention that leap seconds are to be ignored? 

I think this is probably a track-specific thing, so it can go into the `instructions.append.md` file for each track. That's my instinct.

**Comment by habere-et-dispertire** [comment]
Thanks. I tested it in [a raku implementation](https://github.com/exercism/raku/pull/596) of the gigasecond exercise.

**Comment by kytrinyx** [comment]
Alright, it seems like we're good on this one. I'm going to go ahead and merge it, and if anyone has any further concerns, please open a PR to suggest changes.

**Pull request merged**